osidb/models/tracker.py

```diff
@@ -129,7 +130,7 @@ class Meta:
             GinIndex(fields=["acl_read"]),
         ]
 
-    objects = TrackerManager()
+    objects = TrackerManager.from_queryset(CustomQuerySetUpdatedDt)()
```

Comment:
I am thinking about Trackers and `update` usage as we are not an authoritative source for Trackers. I am not sure whether we are even calling `update` for Trackers somewhere in the code, but if we do I think it should not auto updated the `updated_dt` as we are not setting the value for this but Jira/Bugzilla do.

Reply:
Ah, I see. Yeah, so in such cases this would incorrectly update the `updated_dt` as `updated_dt` for trackers needs to always match with the origin BTS system (Jira/Bugzilla).